packages/appeals-service-api/src/services/back-office-v2/formatters/index.js

```diff
@@ -41,7 +42,7 @@ const reduceFormatters =
 	(submittable) =>
 	(formatters, [appealType, dirName]) => ({
 		...formatters,
-		[appealType]: require(`./${dirName}/${submittable}`).formatter
+		[appealType]: require(`./${dirName}/${submittable}`)?.formatter
```

Comment:
I've slung this in because there's no s78 appeal file yet

Reply:
Ah ok this isn't enough, looks like the require panics. Options are 
- Add an arbitrary formatter function for S78 appeals
or, better
- Make this require conditional depending on `fs.existsSync`